lib/matplotlib/axes/_base.py

```diff
@@ -2056,7 +2042,7 @@ def draw(self, renderer=None, inframe=False):
         # if the minimum zorder is negative, start rasterization
         rasterization_zorder = self._rasterization_zorder
         if (rasterization_zorder is not None and
-            len(dsu) > 0 and dsu[0][0] < rasterization_zorder):
+                len(dsu) > 0 and dsu[0][0] < rasterization_zorder):
```

Comment:
I see a bunch of changes like this--did PEP8 change its mind?  I was fairly sure that the original version was what it wanted.